CMakeLists.txt

```diff
@@ -20,14 +24,29 @@ endif()
 if (POLICY CMP0077)
     # Allow CMake 3.13+ to override options when using FetchContent / add_subdirectory.
     cmake_policy(SET CMP0077 NEW)
-endif ()
+endif()
+
+if(CMAKE_VERSION VERSION_GREATER_EQUAL "3.24.0")
+  cmake_policy(SET CMP0135 NEW)
+endif()
```

Comment:
I am happy you add this piece of code, as this removes yet another CMake warning!